fast-logger/System/Log/FastLogger/Date.hs

```diff
@@ -55,7 +58,15 @@ formatDate fmt = formatUnixTime fmt . fromEpochTime
 ----------------------------------------------------------------
 
 -- |  Make 'IO' action which get cached formatted local time.
-newTimeCacher :: TimeFormat -> IO (IO FormattedTime)
-newTimeCacher fmt = mkAutoUpdate defaultUpdateSettings{
+-- Use this to avoid the cost of frequently time formatting by caching an
+-- auto updating formatted time, this cache update every 1 second.
+-- more detail in "Control.AutoUpdate"
+newTimeCache :: TimeFormat -> IO (IO FormattedTime)
+newTimeCache fmt = mkAutoUpdate defaultUpdateSettings{
         updateAction = getTime >>= formatDate fmt
     }
+
+-- | A simple time cache using format @"%d/%b/%Y:%T %z"@
+simpleTimeCache :: IO FormattedTime
+simpleTimeCache = unsafePerformIO $ newTimeCache "%d/%b/%Y:%T %z"
+{-# NOINLINE simpleTimeCache #-}
```

Comment:
I guess that this global cacher can be easily implemented by users. So, I cannot find good reasons to define this global cacher in this package.